src/views/_global/holaplexConstants.ts

```diff
@@ -1,5 +1,37 @@
 export const HOLAPLEX_MARKETPLACE_SUBDOMAIN = `haus`;
 
+// TODO: deprecate
 export const HOLAPLEX_MARKETPLACE_ADDRESS = `9SvsTjqk3YoicaYnC4VW1f8QAN9ku7QCCk6AyfUdzc9t`;
 
+// TODO: deprecate
+export const OPENSEA_MARKETPLACE_ADDRESS = '3o9d13qUvEuuauhFrVom1vuCzgNsJifeaBYDPquaT73Y';
+
+export const AUCTION_HOUSE_ADDRESSES = [
```

Comment:
Can you define an interface for these just so we get some type checking?

Reply:
gotcha did this as part of the utility function work